packages/studio-base/src/components/PanelCatalog/PanelListItem.tsx

```diff
@@ -76,6 +77,12 @@ export function PanelListItem(props: Props): JSX.Element {
     options: { dropEffect: "copy" },
     end: (_item, monitor) => {
       const dropResult = monitor.getDropResult() ?? {};
+
+      // do nothing when the user wants to cancel a dragged panel
+      if (_.isEmpty(dropResult)) {
```

Comment:
Maybe we could just remove this `??` and use `dropResult == undefined` below?

```suggestion
      const dropResult = monitor.getDropResult();

      // do nothing when the user wants to cancel a dragged panel
      if (dropResult == undefined) {
```